### PR TITLE
Add July 6 notes

### PR DESCRIPTION
**Pull request by unho**

### CODE REVIEW

File: 2016-07/july-6.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+
+### Triaging
+
+Some new issues this week.
+
+
+#### Assigned to 2.8
+
+* Popup confirmation always shows up with keyboard submission
```
Comment (julen) [inline 2016-07/july-6.md]: There's no possibility we went through this, as it was filed after the triage meeting.

File: 2016-07/july-6.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+* [Ryan](https://github.com/phlax) (Translate House)
+* [Taras](https://github.com/ta2-1) (Translate House)
+
+
+### Short status and upcoming endeavours report
+
+* 2.8.0b1 was released. The final 2.8.0 will happen before end of July 2016.
+  Bits of `pootle_fs` still need to be landed for that to happen, but it is
+  mostly done.
+* Julen will finish his work on editor using custom font, and then will start
```
Comment (julen) [inline 2016-07/july-6.md]: I didn't mention this; it seems like a copy & paste from the previous notes.

**Comment by julen** [comment]
Thank you @unho, give me some time and I'll propose some minor changes separately.

**Comment by unho** [comment]
@julen Thanks for the review. Already corrected the issues you highlighted.

**Comment by unho** [comment]
Landed to master.

**Pull request closed without merging**